backend/ipmi.pm

```diff
@@ -108,7 +108,7 @@ sub do_start_vm {
     # So keep it for flexibility.
     $self->do_mc_reset if $bmwqemu::vars{IPMI_BACKEND_MC_RESET};
     $self->get_mc_status;
-    $self->restart_host;
+    $self->restart_host if !$bmwqemu::vars{IPMI_DO_NOT_RESTART_HOST};
```

Comment:
better use

```perl
$self->restart_host unless $bmwqemu::vars{IPMI_DO_NOT_RESTART_HOST};
```

Reply:
Hi Oliver, thanks for the detailed guidance. Refined the code change, enlarge the commit description and gave a new push.